core/src/main/scala/org/apache/spark/broadcast/BroadcastManager.scala

```diff
@@ -56,7 +57,10 @@ private[spark] class BroadcastManager(
   private val nextBroadcastId = new AtomicLong(0)
 
   private[broadcast] val cachedValues = {
```

Comment:
You don't need the braces here, but I wouldn't bother unless you're changing this file again